### DISCUSSION TITLE
Preferred Filter Naming Convention

### DISCUSSION THREAD

**Discussion by imikejackson** [BlueQuartzSoftware/DREAM3DNX-Issues#16] [Polls]

**Comment by imikejackson** [comment]
## Complete List of Filters that are Affected.

- OrientationAnalysis,FindAvgCAxesFilter
- OrientationAnalysis,FindAvgOrientationsFilter
- OrientationAnalysis,FindBoundaryStrengthsFilter
- OrientationAnalysis,FindCAxisLocationsFilter
- OrientationAnalysis,FindFeatureNeighborCAxisMisalignmentsFilter
- OrientationAnalysis,FindFeatureReferenceCAxisMisorientationsFilter
- OrientationAnalysis,FindFeatureReferenceMisorientationsFilter
- OrientationAnalysis,FindGBCDFilter
- OrientationAnalysis,FindGBCDMetricBasedFilter
- OrientationAnalysis,FindGBPDMetricBasedFilter
- OrientationAnalysis,FindKernelAvgMisorientationsFilter
- OrientationAnalysis,FindMisorientationsFilter
- OrientationAnalysis,FindSchmidsFilter
- OrientationAnalysis,FindShapesFilter
- OrientationAnalysis,FindSlipTransmissionMetricsFilter
- OrientationAnalysis,FindTriangleGeomShapesFilter
- OrientationAnalysis,GenerateFZQuaternionsFilter
- OrientationAnalysis,GenerateFaceIPFColoringFilter
- OrientationAnalysis,GenerateFeatureFaceMisorientationFilter
- OrientationAnalysis,GenerateGBCDPoleFigureFilter
- OrientationAnalysis,GenerateIPFColorsFilter
- OrientationAnalysis,GenerateQuaternionConjugateFilter
- SimplnxCore,CalculateArrayHistogramFilter
- SimplnxCore,CalculateFeatureSizesFilter
- SimplnxCore,CalculateTriangleAreasFilter
- SimplnxCore,ComputeFeatureRectFilter
- SimplnxCore,ComputeMomentInvariants2DFilter
- SimplnxCore,FindArrayStatisticsFilter
- SimplnxCore,FindBiasedFeaturesFilter
- SimplnxCore,FindBoundaryCellsFilter
- SimplnxCore,FindBoundaryElementFractionsFilter
- SimplnxCore,FindDifferencesMapFilter
- SimplnxCore,FindEuclideanDistMapFilter
- SimplnxCore,FindFeatureCentroidsFilter
- SimplnxCore,FindFeatureClusteringFilter
- SimplnxCore,FindFeatureNeighborsFilter
- SimplnxCore,FindFeaturePhasesBinaryFilter
- SimplnxCore,FindFeaturePhasesFilter
- SimplnxCore,FindLargestCrossSectionsFilter
- SimplnxCore,FindNeighborListStatisticsFilter
- SimplnxCore,FindNeighborhoodsFilter
- SimplnxCore,FindNumFeaturesFilter
- SimplnxCore,FindSurfaceAreaToVolumeFilter
- SimplnxCore,FindSurfaceFeaturesFilter
- SimplnxCore,FindTriangleGeomCentroidsFilter
- SimplnxCore,FindTriangleGeomSizesFilter
- SimplnxCore,FindVertexToTriangleDistancesFilter
- SimplnxCore,FindVolFractionsFilter
- SimplnxCore,GenerateColorTableFilter
- SimplnxCore,GeneratePythonSkeletonFilter
- SimplnxCore,GenerateVectorColorsFilter
- SimplnxReview,FindGroupingDensityFilter
- SimplnxReview,FindLocalAverageCAxisMisalignmentsFilter
- SimplnxReview,FindMicroTextureRegionsFilter
- SimplnxReview,FindSaltykovSizesFilter

**Reply by CodyPrattMME** [reply]
I voted for "compute," but I'm team "determine" until someone changes my mind. 😊

**Reply by imikejackson** [reply]
@CodyPrattMME I'm not completely against that terminology
